Tests/RulesTests+Organization.swift

```diff
@@ -1053,7 +1053,7 @@ class OrganizationTests: RulesTests {
 
             // MARK: lifecycle
 
-            // MARK: Lifeycle
+            // MARK: Lifecycle
```

Comment:
This typo was intentional because the test case is validating that the rule handles typos correctly

```suggestion
            // MARK: Lifeycle
```